pages/start.tsx

```diff
@@ -45,13 +46,20 @@ interface LoginStep1Props {
 }
 
 const LoginStep1: React.FC<LoginStep1Props> = ({ user }) => {
-  const { captureAnalytics } = useAnalytics();
+  const { data: userInfo, isLoading } = useFetchUser(user?.user_metadata.user_name);
 
-  captureAnalytics({
-    title: "User Onboarding",
-    property: "onboardingStep1",
-    value: "visited",
-  });
+  useEffect(() => {
```

Comment:
Added the `captureAnalytics` calls in `useEffect`s so they aren't called multiple times, i.e. only capture analytics once loading is complete.

Same for the other files below.